src/sonic-yang-models/tests/yang_model_tests/test_yang_model.py

```diff
@@ -186,6 +186,10 @@ def initTest(self):
                     about high threshold being lower than low threshold.',
                 'eStr': ['high_threshold should be more than low_threshold']
             },
+            'CRM_WITH_WRONG_THRESHOLD_TYPE': {
+                'desc': 'CRM_WITH_WRONG_THRESHOLD_TYPE must condition failure.',
+                'eStr': self.defaultYANGFailure['None']
```

Comment:
If it is "must condition failure" then estr should be self.defaultYANGFailure['must'] + field name.